libs/pcre2/Config.in

```diff
@@ -0,0 +1,30 @@
+config PCRE2_JIT_ENABLED
+	bool
+	depends on PACKAGE_libpcre2 && (aarch64 || aarch64_be || arm || i386 || i686 || x86_64 || mips || mipsel || mips64 || mips64el || powerpc || powerpc64 || powerpcle || sparc)
+	default y if (arm || i686 || x86_64)
```

Comment:
Remove this. It will fail on the buildbots as luajit needs 32-bit host libraries. Not having libpcre2 will cause long range build failures.

Change it to default n.

Reply:
I would rather see getting the buildbots fixed instead of getting lower performance because just because especially since we're not dealing with high performance devices to begin with.

Reply:
For the time being though, it makes more sense to default to n. Once the buildbots are fixed this can be turned back on by default.

Reply:
I am not sure how this commit is related to luajit. Do you mean that JITs in General cause build problems in 32bit? If so, why does my previous pcre(1) PR not cause problems?

Reply:
https://downloads.openwrt.org/snapshots/faillogs/mips_24kc/packages/luajit/compile.txt

Needs 32-bit gcc and libraries.

Reply:
Looked again. It looks like I'm mistaken. Merging.